spec/features/pages_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'Clicking navbar links', :type => :feature do
+
+  it 'goes to the homepage' do
+    visit '/about'
+    find_link('Home').click
+    expect(page).to have_content 'Connect Customers in Need with Veterans in Skilled Trades'
+  end
+
+  it 'goes to the how page' do
+    visit '/'
+    find_link('How').click
+    expect(page).to have_content 'How It Works:'
+  end
+
+  it 'goes to the about page' do
+    visit '/'
+    find_link('About').click
+    expect(page).to have_content 'For our beloved veterans in the skilled trades,'
+  end
+
+  it 'goes to the Customer Sign-up page' do
+    visit '/'
+    page.find('#customer-sign-up').click
+    expect(page).to have_content 'Customer Registration'
+  end
+
+  it 'goes to the Customer Sign-in page' do
+    visit '/'
+    page.find('#customer-sign-in').click
+    expect(page).to have_content 'Customer Sign In'
+  end
+end
+
+RSpec.describe "Clicking content links on splash page", :type => :feature do
```

Comment:
lets call this session/registration links? and why the separate describe block for actions that seem similar?